### PR TITLE
deb/rpm: move server-centric CLI tools from ceph-common to ceph-base

### PR DESCRIPTION
**Pull request by smithfarm**
Downstream, we've had the `cephfs-*` fsck tools and `ceph-dencoder` in ceph-base for a long time, with no adverse effect.

Not so sure about `radosgw-admin`, which @runsisi asked to also be moved, but it seems pretty server-centric (?)

Fixes: https://tracker.ceph.com/issues/42430

### CODE REVIEW

**Comment by smithfarm** [comment]
https://shaman.ceph.com/builds/ceph/wip-42430/0fe84d08213619e74a6dbb7bb105c8cb04a5aa72/

**Comment by smithfarm** [comment]
@cbodley @mattbenjamin How do you feel about moving `radosgw-admin` from ceph-common to ceph-base?

The rationale for not having it in `ceph-common` is that it introduces extra dependencies to, and increases the "weight" of, what should be a lightweight, client-focused package.

If `ceph-base` is not a good place, would it be worth considering to package it separately, i.e. make a completely new, separate `radosgw-admin` package?

**Comment by smithfarm** [comment]
@cbodley @mattbenjamin Ping

**Comment by smithfarm** [comment]
Any objections to this?

**Comment by liewegas** [comment]
@cbodley @mattbenjamin @yehudasa ping? I'm not sure about radosgw-admin either...

**Comment by cbodley** [comment]
> @cbodley @mattbenjamin How do you feel about moving `radosgw-admin` from ceph-common to ceph-base?
> 
> The rationale for not having it in `ceph-common` is that it introduces extra dependencies to, and increases the "weight" of, what should be a lightweight, client-focused package.
> 
> If `ceph-base` is not a good place, would it be worth considering to package it separately, i.e. make a completely new, separate `radosgw-admin` package?

i do think it would make sense to package radosgw-admin separately. but if moving it to ceph-base is sufficient, i'm fine with that too

**Comment by smithfarm** [comment]
How about I merge this and backport it to nautilus, and follow up with another PR (including a PendingReleaseNote) that splits radosgw-admin into its own package for Octopus?

**Comment by smithfarm** [comment]
Note: this will need some more updates to make sure package upgrades go smoothly. Marking DNM for now.

File: debian/ceph-base.install [hunk 1]
```diff
@@ -3,6 +3,9 @@ lib/systemd/system/ceph-crash.service
 usr/bin/ceph-crash
 usr/bin/ceph-debugpack
 usr/bin/ceph-run
+usr/bin/cephfs-data-scan
```
Comment (tchaikov) [inline debian/ceph-base.install]: this is the `#9` case in https://wiki.debian.org/PackageTransition. shall we update the relation between ceph-base and ceph-common accordingly?

**Comment by liewegas** [comment]
My concern is that any radosgw-admin user is either running on a "client" host with ceph-common on it or on a "server" host with ceph-base + everything else.  After an upgrade, those used to running radosgw-admin on the "client"-style machines will be surprised.  I would put radosgw-admin in the same management category as the ceph CLI, which makes me think leaving it in ceph-common will be somewhat less surprising.  (All of the other tools in this PR are debugging or disaster-recovery tools.)

Maybe we leave it in ceph-common for nautilus, and move it to its own package for octopus?

**Comment by smithfarm** [comment]
unstale

**Pull request closed without merging**